docs/mobile-apps/automated-testing/espresso-xcuitest/espresso-capture.md

```diff
@@ -0,0 +1,39 @@
+---
+id: espresso-capture
+title: Espresso Screenshot Capture
+sidebar_label: Espresso Screenshot Capture
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The Espresso Screenshot Capture functionality allows you to take screenshots of errors to support the visual verification of your debugging process. These screenshots also help to identify any errors or bugs within your application.
+
+
+## Capturing Screenshots
```

Comment:
Just a check-check-double-check (and maybe @kristofmuhi can confirm) but does this work for RDC only or also Emulators?
If it only works for RDC we should
- make this clearer in the docs
- only mention the RDC endpoints and remove the VDC endpoints

If it works on both we should also make that clearer 😄

Reply:
Corrected.